types.ts

```diff
@@ -10,6 +10,19 @@ export type ClientOptions = {
   defaultVariables?: Record<string, string>;
 };
 
+type IsNode = typeof globalThis extends { process: unknown } ? true : false;
```

Comment:
Is there a better way how to define that type?

Reply:
I did not really find a better way!